css/input.css

```diff
@@ -79,6 +79,8 @@
     -moz-text-decoration-color: red;
     -ms-text-decoration-color: red;
     text-decoration-color: red;
+
```

Comment:
When you are doing browser-specific css, you typically add extra lines between the sections of 'duplicate' css. This border color is visually separate from the text-decoration block. It makes sense if you look at the entire invalid css styling.